### PR TITLE
update fcis variable names

### PR DESCRIPTION
**Pull request by knorth55**
- `(R', 2, L, RH, RW)`: roi_cls_ag_seg
- `(R', L, RH, RW)`: roi_cls_seg
- `(R', 2, RH, RW)`: roi_ag_seg
- `(R', RH, RW)`: roi_seg

### CODE REVIEW

File: chainercv/experimental/links/model/fcis/utils/mask_voting.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def _mask_aggregation(
 
 
 def mask_voting(
-        roi_cmask_prob, bbox, roi_cls_prob, size,
```
Comment (yuyu2172) [inline chainercv/experimental/links/model/fcis/utils/mask_voting.py]: How about `roi_seg_prob` --> `seg_prob` and `roi_cls_prob` -> `cls_prob`.

Also, there is an inconsistency in the documentation and the variable names.

**Comment by yuyu2172** [comment]
Can you rename `test_mask_voting.py` too?

**Comment by knorth55** [comment]
I rename variables in `test/` too.

**Comment by yuyu2172** [comment]
LGTM

**Pull request merged**